runtime/descriptor.go

```diff
@@ -19,6 +19,13 @@ import (
 	"reflect"
 )
 
+type fieldDescriptorType int
+
+const (
+    FIELDDESCRIPTOR_RO fieldDescriptorType = iota
```

Comment:
Const names should use the same naming convention as variable names. Also, these should be private to the package, since makeStructFieldDescriptor is not public anyway: fieldDescriptorRO, fieldDescriptorRW